test/e2e/apps/rc.go

```diff
@@ -193,6 +238,16 @@ var _ = SIGDescribe("ReplicationController", func() {
 		ginkgo.By("patching ReplicationController scale")
 		_, err = f.ClientSet.CoreV1().ReplicationControllers(testRcNamespace).Patch(context.TODO(), testRcName, types.StrategicMergePatchType, []byte(rcScalePatchPayload), metav1.PatchOptions{}, "scale")
 		framework.ExpectNoError(err, "Failed to patch ReplicationControllerScale")
+		ginkgo.By("waiting for RC to be modified")
+
+		eventFound = false
+		for watchEvent := range rcWatchChan {
+			if watchEvent.Type == watch.Modified {
+				eventFound = true
+				break
+			}
+		}
+		framework.ExpectEqual(eventFound, true, "failed to find RC %v event", watch.Modified)
```

Comment:
Do we need this?  It seems like the below watch event loop is going to wait until the RC status converges to the new number of replicas, I'm not sure why we need to wait before we wait